src/App.js

```diff
@@ -1,18 +1,43 @@
+/* eslint-disable no-lone-blocks */
+/* eslint-disable react/prefer-stateless-function */
 import './style/style.scss';
-import React, { Component } from 'react';
-import Calculator from './components/calculator';
+import { Link, Route, Routes } from 'react-router-dom';
+import React from 'react';
 import './components/logic/calculate';
 import './components/logic/operate';
+import Cal from './components/CalculatorPart';
+import Quote from './components/Quote';
+import Home from './components/Home';
 
-// eslint-disable-next-line react/prefer-stateless-function
-class App extends Component {
-  render() {
-    return (
-      <div className="App">
-        <Calculator />
-      </div>
-    );
-  }
+function App() {
+  return (
+    <>
+      <nav className="navbar">
+        <div className="Title-name">
+          MATH MAGICIANS
+        </div>
+        <div className="nav-links">
+          <Link to="/Home"> Home </Link>
+          <Link to="/Calculator"> Calculator </Link>
+          <Link to="/Quote"> Quote </Link>
+        </div>
+      </nav>
+      <Routes>
+        <Route path="/Home" element={<Home />} />
+        <Route path="/Calculator" element={<Cal />} />
+        <Route path="/Quote" element={<Quote />} />
+      </Routes>
+    </>
+  );
 }
 
 export default App;
+
+{ /* <nav className="navbar">
+      <div className="Title-name">MATH MAGICIANS</div>
+      <div className="nav-links">
+        <a href="/Home">HOME</a>
+        <a href="/Calculator">Calculator</a>
+        <a href="/Quote">QUOTE</a>
+      </div>
+    </nav> */ }
```

Comment:
- [ ] I noticed that you comment code here.  :cry:  Removing commented code not only makes it harder to find later, but it also makes it so people in the future don't know it existed before. For finding the code, there are [tools and git commands](http://stackoverflow.com/q/278192/971592) to help you look at the history of a file. Please remove all commented code and unnecessary files from your code base :smile: